package.json

```diff
@@ -31,5 +31,6 @@
   },
   "dependencies": {
     "axios": "^0.27.2"
-  }
+  },
+  "-t":"5000"  
```

Comment:
This is wrong mister, you should edit the script "test" and add the -t 5000

Reply:
To be more accurate, in line 7 of this file

Reply:
Ok, thank you!!!, but, which one of the following must I apply?:
1. "test": "mocha -t 5000" or
2. "test": "-t 5000"

Thanks in advance!

Reply:
The first one, is an additional parameter to the execution!